tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -77,6 +77,55 @@ def environment_length_bytes(e: dict) -> int:
     return string_length_bytes(serialized_environment)
 
 
+class TestPartialARNMatching:
+    @markers.aws.validated
+    def test_update_function_configuration_full_arn(self, create_lambda_function, aws_client):
+        function_name = f"fn-{short_uid()}"
+        create_response = create_lambda_function(
+            handler_file=TEST_LAMBDA_PYTHON_ECHO,
+            func_name=function_name,
+            runtime=Runtime.python3_12,
+            MemorySize=256,
+            Timeout=5,
+        )
+
+        aws_client.lambda_.get_waiter("function_active_v2").wait(FunctionName=function_name)
```

Comment:
FYI: The fixture does the waiting automatically, but I think it's a good idea to make it explicit here 👍